.github/workflows/main.yml

```diff
@@ -205,22 +207,29 @@ jobs:
         echo "::remove-matcher owner=sphinx::"
       if: matrix.publish
 
-    - name: Validate
+    - name: Validate (Linux)
       run: |
         echo "::add-matcher::./.github/workflows/main/problemMatchers/validateRelease.json"
-        ./config/validateRelease.py --archive ${{ env.GAFFER_BUILD_NAME }}.tar.gz ${{ env.GAFFER_VALIDATE_EXTRA_FLAGS }}
+        ./config/validateRelease.py --archive ${{ env.GAFFER_BUILD_NAME }}.${{ env.PACKAGE_EXTENSION }} ${{ env.GAFFER_VALIDATE_EXTRA_FLAGS }}
         echo "::remove-matcher owner=validateRelease::"
-      if: matrix.publish
+      if: matrix.publish && runner.os != 'Windows'
+
+    - name: Validate (Windows)
+      run: |
+        echo "::add-matcher::./.github/workflows/main/problemMatchers/validateRelease.json"
+        python ./config/validateRelease.py --skipDocs --archive ${{ env.GAFFER_BUILD_NAME }}.${{ env.PACKAGE_EXTENSION }} ${{ env.GAFFER_VALIDATE_EXTRA_FLAGS }}
+        echo "::remove-matcher owner=validateRelease::"
+      if: matrix.publish && runner.os == 'Windows'
```

Comment:
Fixed in 677c431eba947cead395c5724276a7c3ad7d9828. It might be nice to have `GAFFER_VALIDATE_EXTRA_FLAGS` in `main.yml` instead, but I couldn't find any information about adding environment variables to a build matrix.

Reply:
Good point - I'm not sure why it was historically an environment variable. We could just use `{{ matrix.validateFlags }}` instead, following the example of `{{ matrix.testArguments }}`?